nebula.dme

```diff
@@ -3460,5 +3457,9 @@
 #include "maps\~mapsystem\maps_unit_testing.dm"
 #include "maps\~unit_tests\unit_testing.dm"
 #include "mods\_modpack.dm"
+#include "mods\persistence\modules\client\preferences.dm"
```

Comment:
Don't include modpack stuff in here. Things in the modpack have to go under mods/presistence/persistence.dme otherwise things are gonna get really wonky with nebula.dme being loaded before persistence.dme.